packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart

```diff
@@ -191,6 +35,16 @@ class BuildRunnerWebCompilationProxy extends WebCompilationProxy {
   BuildImpl _builder;
   PackageUriMapper _packageUriMapper;
 
+  @override
+  Future<DaemonHandle> daemon({
```

Comment:
TBD what the daemon command looks like exactly, but I imagine we need the stream of build results, and a handle to pass a List<String> or List<AssetId> to the daemon